tests/permissions/scripts/setup_access.py

```diff
@@ -2,9 +2,9 @@
 import hubmodule.models as hm
 from laminhub_rest.core.db import DbRoleHandler
 
-full_access = ln.models.Space(name="full access", uid="00000001").save()
-select_access = ln.models.Space(name="select access", uid="00000002").save()
-no_access = ln.models.Space(name="no access", uid="00000003").save()
+full_access = ln.models.Space(name="full access", uid="00000001").save()  # type: ignore
```

Comment:
What is the problem here? Why type error?

Reply:
Because the user-facing constructor doesn't accept `uid` as an arg.